test/unit/karma.conf.js

```diff
@@ -42,23 +42,26 @@ module.exports = function (config) {
             'test/unit/testUtil.js',
             'test/unit/mocks.js',
             'test/unit/test-main.js',
-            { pattern: 'test/unit/spec/**/*.js', included: false },
+            { pattern: 'test/unit/spec/**/*.js', included: false, nocache: true },
             { pattern: 'test/testConfig.json', included: false, nocache: true },
             { pattern: 'test/*.tok', included: false, nocache: true },
             { pattern: 'test/data/**/*', included: false },
             // JS files
             'kbase-extension/static/narrative_paths.js',
-            { pattern: 'kbase-extension/static/**/*.js', included: false },
-            { pattern: 'nbextensions/appcell2/widgets/tabs/*.js', included: false },
+            { pattern: 'kbase-extension/static/**/*.js', included: false, nocache: true },
+            { pattern: 'nbextensions/**/*.js', included: false, nocache: true },
```

Comment:
When running tests in TDD mode, this ensures files aren't cached, so you can make changes and rerun the tests to take account of those changes.